### PR TITLE
Admin list users and delete user

### PR DESCRIPTION
**Pull request by danielmdavis**
- Created admin page for removing users

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class UsersController < ApplicationController
+
```
Comment (AL6981) [inline app/controllers/users_controller.rb]: You need to add a bit more to this controller.   before_action :authorize_user and before_action :authorize_admin, and create the methods in the controller to ensure correct authorization. Refer to this link for details. https://learn.launchacademy.com/teams/boston-20/curricula/on-campus-boston-20/lesson_groups/weeks_7_&_8:_group_project/lessons/devise-authorization

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<div class="row user-page">
+  <div class="columns medium-10">
+    <h1>All Users</h1>
+
+    <div class="row">
```
Comment (AL6981) [inline app/views/users/index.html.erb]: You have divs with h tags, but that's it. Are these supposed to be here?

File: config/routes.rb [hunk 3]
```diff
@@ -20,5 +20,7 @@
 
   resources :spots, only: [:index, :show, :create, :new, :edit, :update]
 
+  resources :users, only: [:index, :destroy]
```
Comment (AL6981) [inline config/routes.rb]: This is the second time you've listed resources users, with index listed in both. What's the functionality you're looking for here?

Comment (garabedium) [reply config/routes.rb]: This is used for an API call (returning spots for a given user) on a branch that hasn't merged yet.

File: app/views/users/index.html.erb (hunk 2 above)
Comment (garabedium) [reply app/views/users/index.html.erb]: There's more in that file. It has a list of all users and the ability to delete those users.

**Pull request merged**